<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VisibleSettingDlg</class>
 <widget class="QDialog" name="VisibleSettingDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>384</width>
    <height>145</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QRadioButton" name="radioButton_Hide">
   <property name="geometry">
    <rect>
     <x>125</x>
     <y>81</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Hide</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_Show">
   <property name="geometry">
    <rect>
     <x>193</x>
     <y>81</y>
     <width>67</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Show</string>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>109</y>
     <width>176</width>
     <height>27</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>366</width>
     <height>62</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QCheckBox" name="cBox_Scalar">
      <property name="text">
       <string>Scalar</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QComboBox" name="comboBox_Scalar"/>
    </item>
    <item row="0" column="2" colspan="2">
     <widget class="QLineEdit" name="lineEdit_Scalar"/>
    </item>
    <item row="1" column="0">
     <widget class="QCheckBox" name="cBox_Dim">
      <property name="text">
       <string>Dim</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QComboBox" name="comboBox_Dim">
      <item>
       <property name="text">
        <string>Z</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>X</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Y</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QSpinBox" name="sBox_LowDim">
      <property name="minimum">
       <number>-9999999</number>
      </property>
      <property name="maximum">
       <number>9999999</number>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QSpinBox" name="sBox_HighDim">
      <property name="minimum">
       <number>-9999999</number>
      </property>
      <property name="maximum">
       <number>9999999</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>VisibleSettingDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>227</x>
     <y>101</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>110</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>VisibleSettingDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>236</x>
     <y>101</y>
    </hint>
    <hint type="destinationlabel">
     <x>245</x>
     <y>110</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
